packages/api-server/src/methods/modules/eth.ts

```diff
@@ -1005,7 +1005,7 @@ export class Eth {
       }
 
       const fromBlockNumber: U64 = await this.blockParameterToBlockNumber(
-        filter.fromBlock || "latest"
+        filter.fromBlock || "earliest"
```

Comment:
It appears that `filter.fromBlock` has different default values for different cases. It would be helpful to me that add a  code comment with a link to the standard documentation.

I only searched out [`web3.eth.getPastLogs`](https://web3js.readthedocs.io/en/v1.2.11/web3-eth.html#id102)

Reply:
I'm not sure if it's a standard document @RetricSu

Reply:
pls note that this is only a temp fix to show the full log and keep the behavior compatible with ethereu,. the standard fromBlock should be "latest". https://eth.wiki/json-rpc/API, see `eth_newFilter` section.

in order to use the more clear code with the latest tag, there requires another refactor pr, which we don't have the capacity for now, and the priority is not very high, should we will do it later.

Reply:
> pls note that this is only a temp fix to show the full log and keep the behavior compatible with ethereum,. the standard fromBlock should be "latest". https://eth.wiki/json-rpc/API, see `eth_newFilter` section.
> 
> in order to use the more clear code with the latest tag, there requires another refactor pr, which we don't have the capacity for now, and the priority is not very high, so we will do it later.

Thanks. Resolve this conversation after commenting this note @classicalliu